ruby/README.md

```diff
@@ -0,0 +1,79 @@
+# Ruby
+
+- Avoid conditional modifiers (lines that end with conditionals).
+- Avoid multiple assignments per line (`one, two = 1, 2`).
+- Avoid organizational comments (`# Validations`).
+- Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
+  instead to emphasize code branches.
+- Avoid bang (!) method names. Prefer descriptive names. [#122]
+- Name variables created by a factory after the factory (`user_factory` creates
+  `user`).
+- Prefer nested class and module definitions over the shorthand version
+- Prefer `detect` over `find`.
+- Prefer `select` over `find_all`.
+- Prefer `map` over `collect`.
+- Prefer `reduce` over `inject`.
+- Prefer `&:method_name` to `{ |item| item.method_name }` for simple method
+  calls.
+- Use `_` for unused block parameters.
+- Prefix unused variables or parameters with underscore (`_`).
+- Suffix variables holding a factory with `_factory` (`user_factory`).
+- Use a leading underscore when defining instance variables for memoization.
+- Use `%()` for single-line strings containing double-quotes that require
+  interpolation.
+- Use `?` suffix for predicate methods.
+- Use `def self.method`, not `class << self`.
```

Comment:
Would the caveat here be if you are defining many class methods? I think I've seen this around the code base

Reply:
Yeah.

Reply:
@dgalarza what do you think about updating this to `prefer / avoid` over the stronger `use / not` language? I personally feel like there are times `class << self` is a valid pattern and the stronger `use / not` makes it sound like you should never use it.

Reply:
I'm down.